deployment/enterprise/.env.template

```diff
@@ -215,3 +215,14 @@ VECTOR_TILES_STYLE_URL=https://tiles-ee.merginmaps.com//styles/default.json
 #QGIS_EXTRACTOR_TIMEOUT=60
 
 #OVERVIEW_MAX_FILE_SIZE=1048576 # 1MB
+
+### SSO ################################################################################################################
+SSO_ENABLED=True
```

Comment:
I'd say this is optional feature, I would turn it off

Reply:
We should also disable `MAPS_ENABLED=True` as optional feature?